src/ui/components/ProposalCallInfo.tsx

```diff
@@ -1,25 +1,78 @@
 import React from 'react';
-import {View} from 'react-native';
+import {StyleSheet} from 'react-native';
 import {Paragraph, Card, Caption} from '@ui/library';
 import {Padder} from '@ui/components/Padder';
-import type {DemocracyProposal, DemocracyReferendum} from 'src/api/hooks/useDemocracy';
+import type {DemocracyProposal, DemocracyReferendum, ProposalSubCall} from 'src/api/hooks/useDemocracy';
+import {standardPadding} from '@ui/styles';
 
 type ProposalInfoProps = {
   proposal: DemocracyProposal | DemocracyReferendum;
 };
 
 export function ProposalCallInfo({proposal}: ProposalInfoProps) {
```

Comment:
@Ekaanth 
You can use this component to display the call params of the motion proposals.

Reply:
Yes, @ishrak-imam I'm using this component for proposals.